docs/schema.md

```diff
@@ -15,7 +16,7 @@ Example:
 }
 
 // bafy...dialog
-encrypt({
+{
```

Comment:
I don't think it's worth wrapping with `encrypt()` when _every_ block is encrypted.

Reply:
I’m not sure I understood it correctly.
Does this mean I can access the dialogs and messages without needing to decrypt them again? So I just need the CID and fetch it from Storacha?
If that’s the case, does it mean anyone with the (dialog/messages) CID could access the data?

Reply:
No, the only thing that has changed here is that we encrypt the root block in addition to the others.